examples/workdir/inventory/hosts_combined_example.yml

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
Note: hosts_combined_example file is created for review purposes only and
  includes following proposed changes:
   * added "airship-openstack-control-workers", "airship-ucp-workers"
     and "airship-openstack-compute-workers"  host groups that are
     necessary for airship deployment.
     - for CI,
        "airship-openstack-control-workers": set hosts to first two workers
        "airship-ucp-workers": set hosts to first two workers
        "airship-openstack-compute-workers": set hosts to third worker
   * rename 'osh-deployer' to 'socok8s-deployer'
     - since deployer can be used for osh and airship deployment
   * added "all" and "airship-workers" host groups
   * Questions:
     Q1: does deploy_osh role make use of something similar to
         "airship-openstack-control-workers"  or
         "airship-openstack-compute-workers" ? Do they need to be renamed?